cvmfs/sync_mediator.cc

```diff
@@ -972,6 +1014,12 @@ void SyncMediator::AddUnmaterializedDirectory(SharedPtr<SyncItem> entry) {
 }
 
 void SyncMediator::AddDirectory(SharedPtr<SyncItem> entry) {
+  if (entry->filename() == ".cvmfsbundles") {
```

Comment:
We should avoid using the string literal. Can we check instead for `IsBundleSpec()`?